rosidl_adapter/rosidl_adapter/parser.py

```diff
@@ -459,6 +459,9 @@ def parse_message_string(pkg_name, msg_name, message_string):
     for line in lines:
         line = line.rstrip()
 
+        # replace tabs with spaces
+        line = line.replace('\t', ' ')
```

Comment:
Is this relevant to supporting hex values?

Reply:
I think this is pulling #557 into this PR.  I'll suggest dropping it from this one, so we can consider them separately.

Reply:
dropped.